apps/bare-expo/ios/Podfile

```diff
@@ -53,6 +53,8 @@ target 'BareExpo' do
   post_install do |installer|
     installer.target_installation_results.pod_target_installation_results.each do |pod_name, target_installation_result|
       target_installation_result.native_target.build_configurations.each do |config|
+        config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
```

Comment:
Do you think it would make sense to add comment here (and maybe optionally in `build-detox-ios.sh`) why do we add this setting?